llvm/test/CodeGen/AMDGPU/uncalled-local-functions.ll

```diff
@@ -0,0 +1,11 @@
+; RUN: llc -O0 -march=amdgcn < %s | FileCheck %s
+
```

Comment:
Also should have some tests for functions which are not dead, but forced alive with some form of address capture. We need to verify we produce sensible resource results in those cases